m/mahaleo.md

```diff
@@ -0,0 +1,11 @@
+# Mahaleo
+
+![Mahaleo](mahaleo.jpg)
+
+Mahaleo is a folk-pop band from Madagascar that is widely viewed as the most popular Malagasy group of all time. The band was founded by Dama (Rasolofondraosolo Zafimahaleo) with six of his classmates after first performing together during the rotaka student protests at their high school on 13 May 1972. Mahaleo's lyrics draw upon the indirect language of traditional hainteny and ohabolana (Malagasy poetry and proverbs) to expose contemporary political and social issues and invite listeners to identify their own solutions.
+
+Source:[wikipedia](https://en.wikipedia.org/wiki/Mahaleo)
+
+**Genre:** Folk pop
+
+**Artist homepage:** [Mahaleo](https://en.wikipedia.org/wiki/Mahaleo#/media/File:Mahaleo_super_group_of_musicians_from_Madagascar_in_2007.jpg)
```

Comment:
This should go to their homepage, or a page describing them

Reply:
I've changed the link, please check.
 Thank you